static/styles/night_mode.scss

```diff
@@ -460,6 +460,10 @@ on a dark background, and don't change the dark labels dark either. */
         stroke: hsl(0, 0%, 100%);
     }
 
+    .history-limited-box {
+        background-color: rgba(0, 0, 0, 0.2);
+    }
```

Comment:
I didn't update this; probably another place where using the alert-box styling would save complexity.